lib/widgets/Buttons.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+
+// black button with white text
+class CButton extends StatelessWidget {
```

Comment:
Just wondering why this is called CButton?

Reply:
stands for 'Carriage button'. not a great name but I was assuming this will be the 'standard' button used in designs.